lib/commands/create-migration.js

```diff
@@ -5,18 +5,29 @@ const log = require('db-migrate-shared').log;
 const mkdirp = Promise.promisify(require('mkdirp'));
 const fs = require('fs');
 const stat = Promise.promisify(fs.stat);
+const writeFile = Promise.promisify(fs.writeFile);
 const yargs = require('yargs');
 const util = require('util');
+const path = require('path');
 
 async function createMigrationDir (dir) {
   const res = await stat(dir).catch(_ => {
     return { err: true };
   });
   if (res && res.err === true) {
-    return mkdirp(dir);
+    await mkdirp(dir);
   }
 
-  return Promise.resolve();
+  // Create migrations/package.json to ensure migration files are
+  // executed as CJS and not ESM
+  // https://github.com/db-migrate/node-db-migrate/issues/721
+  const packageJsonPath = path.resolve(dir, 'package.json')
+  await stat(packageJsonPath).catch(async (err) => {
```

Comment:
this wont be enough. there are some people already having a package.json in the migrations folder for utilities they use for those. so when it exists it needs to be required and wrote back with the config added.

Reply:
good catch 👍🏿